src/gwt/acesupport/acemode/stan_highlight_rules.js

```diff
@@ -1,134 +1,186 @@
+    var distributionList = "(~)(\\s*)(bernoulli|bernoulli_logit|beta|beta_binomial|binomial|binomial_logit|categorical|categorical_logit|cauchy|chi_square|dirichlet|double_exponential|exp_mod_normal|exponential|frechet|gamma|gaussian_dlm_obs|gumbel|hypergeometric|inv_chi_square|inv_gamma|inv_wishart|lkj_corr|lkj_corr_cholesky|logistic|lognormal|multi_gp|multi_gp_cholesky|multi_normal|multi_normal_cholesky|multi_normal_prec|multi_student_t|multinomial|neg_binomial|neg_binomial_2|neg_binomial_2_log|normal|ordered_logistic|pareto|pareto_type_2|poisson|poisson_log|rayleigh|scaled_inv_chi_square|skew_normal|student_t|uniform|von_mises|weibull|wiener|wishart)\\b";
```

Comment:
The way Stan currently works is it translates the sampling statement
```
y ~ normal(mean, sd)
```
to the function
```
normal_lpdf(y, mean, sd)
```
So `normal(` should be highlighted only when after a `~`, otherwise its `normal_lpdf`.